src/client/core/evaluator.ts

```diff
@@ -0,0 +1,50 @@
+import { typed, create, all, factory } from 'mathjs'
+import * as checker from './../../types/checker'
+
+const config = {}
+const factories = {
+  ...all,
+  // Override for string comparison
+  createEqual: factory(
+    'equal',
+    [],
+    () => (a: string | number, b: string | number) => {
+      return a === b
+    }
+  ),
+  createUnequal: factory(
+    'unequal',
+    [],
+    () =>
+      function unequal(a: string | number, b: string | number) {
+        return a !== b
+      }
+  ),
+  // Custom if-else function
+  createIfElse: factory('ifelse', [], () =>
+    typed('ifelse', {
+      'boolean, string | number, string | number': (
+        condition: boolean,
+        a: string | number,
+        b: string | number
+      ) => (condition ? a : b),
+    })
+  ),
+}
+export const math = create(factories, config)
+
+export function Operation(
```

Comment:
nitpick - it's not a class name, so..
```suggestion
export function operation(
```